Docs for the deployment changes templates (#535)
OctopusDeploy/docs: docs/deployment-process/releases/deployment-notes.md

  #{/unless}
#{/each}
```

> ## Deployment changes templates {#Templates}

Reviewer: ```suggestion
## Deployment Changes Templates {#Templates}
```

> Starting from `2019.9.0` you can specify a template for deployment changes output. The output of the template is used in the portal for rendering the changes on both the deployment preview and on the deployment details.

Reviewer: ```suggestion
Starting from `2019.9.0` you can specify a template for deployment changes output. The output of the template is used in the portal to render the changes on both the deployment preview and on the deployment details.
```

```Deployment contained releases:<br/>
#{Octopus.Deployment.ChangesMarkdown | MarkdownToHtml}
```

Use this method you would then see exactly the same content in the portal for the deployment preview, on the deployment details, and on the email that gets sent.

All variables available during the deployment are available to the template, with the exception of the machine scoped variables. This is because the templates is applied at the deployment level when the deployment is created, not at the machine level as each target is deployed to. There is a new variable available if you want target information, see below for details.

All environment and tenant related variables are available to the template.

The default template, when the field in the project settings is left blank, is as follows

```
#{each release in Octopus.Deployment.Changes}
**Release #{release.Version}**

#{release.ReleaseNotes}
#{each workItem in release.WorkItems}
- [#{workItem.Id}](#{workItem.LinkUrl}) - #{workItem.Description}
#{/each}
#{/each}
```

> ### Deployment changes and targets

Reviewer: ```suggestion
### Deployment Changes and Targets
```
